.teamcity/_self/projects/WPComTests.kt

```diff
@@ -263,7 +261,12 @@ fun gutenbergPlaywrightBuildType( targetDevice: String, buildUuid: String, atomi
 			)
```

Comment:
A few lines above, we still keep a checkbox for Atomic. It seems to me that we don't really need this if we're removing the checkbox for gutenberg edge? (They would be used in similar ways, so it seems like we'd either keep or remove them together)

Reply:
You're right, the checkbox param to specify env.TEST_ON_ATOMIC is unnecessary now. I will remove it in a revision.